### ISSUE TITLE
PaperVisuals does not allow for any font color other than dark colors

### ISSUE THREAD

**Issue by Codubreku** [space-wizards/space-station-14#43834]
### Description

I am making what essentially is a chalkboard entity that uses paper visuals to display its text. The plan was for it to have a dark gray chalkboard and the paper ui was going to reflect this. However, fontAccentColor, the function that shifts the style of the font does not completely change the font color as I naively hoped. The best I can get, at its brightest, is a very dark-grey. This is a large limitation, as we cannot have darker colored paper or "paper" objects with easily readable font by default.

[Note: I am talking in terms of modifiable default colors for a paper entity. I know [color=x] exists but that is un-ideal when I am making this for players and not specifically myself.]

#ff0000 | Hot Red
<img width="171" height="64" alt="Image" src="https://github.com/user-attachments/assets/925eaaf4-cf94-4fc4-873c-06b351030a7f" />
#ffffff | pure white
<img width="174" height="71" alt="Image" src="https://github.com/user-attachments/assets/0df416d6-fc6a-4d97-b1db-a5023697371c" />

### Reproduction

Steps to reproduce: 
1. open paper.yml
2. add to (picking randomly) office paper's PaperVisuals component the following: fontAccentColor: "#ffffff"
3. Modify the background image to be pure black.
